File: Java/1876. Substrings of Size Three with Distinct Characters.java
class Solution {
    public int countGoodSubstrings(String s) {
        int res = 0, i =0, j =3;
        while(j<=s.length()){
            String window = s.substring(i,j);
            Set<Character> set = new  HashSet<>();
            for (char ch : window.toCharArray()){
                set.add(ch);
            }
            if (set.size() == 3){
                res++;
            }
            i++;
            j++;
        }
        return res;
    }
}

File: Java/1876. Substrings of Size Three with Distinct Characters.py
class Solution:
    def countGoodSubstrings(self, s: str) -> int:
        res = 0
        i = 0
        j = 3
        while j<=len(s):
            window =  s[i:j]
            if (len(set(window)) == 3):
                res+=1
            i+=1
            j+=1
        return res